linux_os/guide/system/accounts/accounts-restrictions/root_logins/use_pam_wheel_group_for_su/bash/shared.sh

```diff
@@ -1,8 +1,15 @@
-# platform = multi_platform_ubuntu
+# platform = multi_platform_ubuntu,multi_platform_sle
 {{{ bash_instantiate_variables("var_pam_wheel_group_for_su") }}}
 
 PAM_CONF=/etc/pam.d/su
 
+if [ $(getent group ${var_pam_wheel_group_for_su}) ]; then
+  # group exists
+   groupdel -f ${var_pam_wheel_group_for_su}
+fi
+groupadd -f ${var_pam_wheel_group_for_su}
+
+
```

Comment:
Isn't this handled by [ensure_pam_wheel_group_empty](https://github.com/ComplianceAsCode/content/tree/master/linux_os/guide/system/accounts/accounts-restrictions/root_logins/ensure_pam_wheel_group_empty) already?

Reply:
Hello @wokis 

1/The rule https://github.com/ComplianceAsCode/content/tree/master/linux_os/guide/system/accounts/accounts-restrictions/root_logins/ensure_pam_wheel_group_empty - does not have ansible part. 
According to this PR I am covering this part. 

2/The rule is a part of ubuntu profile, and I am contribute for SLE 12/15.

Have a nice day
Rumen

Reply:
> Hello @wokis
> 
> 1/The rule https://github.com/ComplianceAsCode/content/tree/master/linux_os/guide/system/accounts/accounts-restrictions/root_logins/ensure_pam_wheel_group_empty - does not have ansible part. According to this PR I am covering this part.
> 
> 2/The rule is a part of ubuntu profile, and I am contribute for SLE 12/15.
> 

As wokis mentioned, you should use the rule `ensure_pam_wheel_group_empty` together with `use_pam_wheel_group_for_su` otherwise you won't be checking through oval if the group is empty and the things you are adding to bash and ansible won't be needed here.